### PR TITLE
MINT-5788 Update fast api template with latest changes

### PR DESCRIPTION
**Pull request by gyandeeps**
https://hingehealth.atlassian.net/browse/MINT-5788

Update the template to use everything latest

### CODE REVIEW

File: .github/workflows/aptible-pentest-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: pentest-deploy
```
Comment (ak47) [inline .github/workflows/aptible-pentest-deploy.yml:1]: I think we can drop these Aptible deploys, Any future services will go straight to AWS

Comment (gyandeeps) [reply .github/workflows/aptible-pentest-deploy.yml:1]: i like it.

File: .github/workflows/aptible-production-deploy.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+name: production-deploy
```
Comment (ak47) [inline .github/workflows/aptible-production-deploy.yml:1]: nuke this?

Comment (gyandeeps) [reply .github/workflows/aptible-production-deploy.yml:1]: i dont know if its too soon. maybe i should?

File: Makefile [hunk 3]
```diff
@@ -14,6 +18,41 @@ aptible-deploy:
 		--private-registry-password=`aws ecr get-login-password --region us-east-1` \
 		DD_VERSION=${GIT_SHA}
 
+# Deploy to aptible by pulling the latest image from private docker registry
+aptible-deploy-latest:
```
Comment (ak47) [inline Makefile:22]: these `aptible-*` commands can be dropped

File: .github/workflows/aptible-production-deploy.yml (hunk 2 above)
Comment (ak47) [reply .github/workflows/aptible-production-deploy.yml:1]: I think if we launched a new service on Aptible at this point, we'd have some serious explaining to do

Comment (gyandeeps) [reply .github/workflows/aptible-production-deploy.yml:1]: true... let me nuke anything which says aptible

Comment (gyandeeps) [reply .github/workflows/aptible-production-deploy.yml]: done

**Review by rsbabcock — APPROVED** [review]

File: app/app/tests/unit/core/test_health_checks.py [hunk 4]
```diff
@@ -35,16 +35,19 @@ def test__database__health(session, expected_value: bool):
     assert is_database_available(session) == expected_value
 
 
-@pytest.mark.asyncio
-@pytest.mark.parametrize("return_value", [b"PONG"])
-async def test__redis__health(async_session):
-    assert await is_redis_available(async_session)
+# @pytest.mark.asyncio
```
Comment (ak47) [inline app/app/tests/unit/core/test_health_checks.py:38]: why comment this out?

File: pyproject.toml [hunk 5]
```diff
@@ -1,51 +1,57 @@
...
 description = "FastAPI template service."
 authors = ["<your-name-here>@hingehealth.com>"]
 
 [tool.poetry.dependencies]
-python = "^3.9"
-pip = "^21.1.1"
-aioredis = "*"
+python = "^3.10"
+pip = "^22.3.1"
+aioredis = "^2.0.1"
```
Comment (ak47) [inline pyproject.toml:10]: When I updated `aioredis` on previous PR attempt, I needed to update `app/app/core/health/checks.py`
wondering why I'm not seeing that change on this PR 🤔 

https://github.com/hinge-health/fastapi-template/pull/26/files#diff-f8035c5651b9b2bc9905e560db938ea09d243e03d3be463bf45d882ef57ff931

Comment (gyandeeps) [reply pyproject.toml:10]: I saw those changes are already in.

File: app/app/tests/unit/core/test_health_checks.py (hunk 4 above)
Comment (gyandeeps) [reply app/app/tests/unit/core/test_health_checks.py:38]: I dont think it returns ping/pong back anymore.

**Review by ak47 — APPROVED** [review]

File: app/app/tests/unit/core/test_health_checks.py (hunk 4 above)
Comment (ak47) [reply app/app/tests/unit/core/test_health_checks.py:38]: I have different code for this on eligibility, thinking we should merge and try a test app.

I have my open PR that will either need to be closed or resolved.

**Pull request merged**